data/SandipPalit/events/2023-08-05-Nomoskar-Kolkata.json

```
@@ -0,0 +1,14 @@
+{
+  "userStatus": "speaking",
+  "speakerDetails": "Introducing Postman Community Kolkata",
```

It should be `speakingTopic` instead. 

```suggestion
  "speakingTopic": "Introducing Postman Community Kolkata",
```